openmetadata-service/src/main/java/org/openmetadata/service/workflows/searchIndex/PaginatedEntityTimeSeriesSource.java

```diff
@@ -145,7 +146,7 @@ public void updateStats(int currentSuccess, int currentFailed) {
   private ListFilter getFilter() {
     ListFilter filter = new ListFilter(null);
     if (ReindexingUtil.isDataInsightIndex(entityType)) {
-      filter.addQueryParam("entityFQNHash", entityType);
+      filter.addQueryParam("entityFQNHash", FullyQualifiedName.buildHash(entityType));
     }
     return filter;
```

Comment:
Maybe instead of doing this if, we could create a new constructor that we use where we need to pass the filter, what do you think?

```
  public PaginatedEntityTimeSeriesSource(String entityType, int batchSize, List<String> fields, ListFilter filter) {
    this.entityType = entityType;
    this.batchSize = batchSize;
    this.fields = fields;
    this.stats
        .withTotalRecords(getEntityTimeSeriesRepository().getTimeSeriesDao().listCount(filter))
        .withSuccessRecords(0)
        .withFailedRecords(0);
  }
  ```